### PR TITLE
[DOCS-431] Documenting Super LAN Connect App--DRAFT--DO NOT MERGE

### PR DESCRIPTION
**Pull request by rajkaramchedu**
This is a placeholder draft for documenting Super LAN Connect app. Dev team hasn't converged yet on what the changes and the impact will be. Nevertheless @workingmonk @jimmyjames comments welcome.

### CODE REVIEW

File: cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Super LAN Connect
+=================
+
+The primary goal of Super LAN Connect app is to minimize the complexity of LAN device discovery. 
+
+The SmartThings platform will discover a LAN-connected or a cloud-connected device only when a service manager SmartApp for that specific device is present. 
+This means that if you want to integrate multiple LAN devices, such as a Samsung SmartCam and a Bose Speaker, into SmartThings network then you will need multiple service manager SmartApps, i.e., a separate service manager SmartApp for each LAN device.  
+On the contrary, the platform does not have any such service manager SmartApp requirement for a ZigBee or a Z-Wave device.
+
+The new Super LAN Connect app eliminates the service manager SmartApp requirement for LAN-connected devices, thereby making for a much smoother and quicker LAN device discovery.
```
Comment (jimmyjames) [inline cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst]: Should say eliminates the Service Manager SmartApp requirement for *some* LAN-connected devices (not all)

File: cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Impact On The Developer
+-----------------------
+
+If you have made any customizations to either your service manager SmartApp or to your Device Handler, then your LAN device integration will be impacted.
+See below.
+
+If your LAN device is **not** in the Super LAN Connect app-supported devices list below, then you are not impacted.
+
+If your LAN device is in the Super LAN Connect app-supported device list below, then:
```
Comment (rajkaramchedu) [inline cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst]: yes.

**Comment by rajkaramchedu** [comment]
@workingmonk please review.
@jimmyjames I see merge conflicts. Let's talk off line, thanks!

File: cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+========== ======================================
+LAN Device Super LAN Connect-supported LAN Device      
+========== ======================================
+Samsung    Samsung speakers  
+Sonos	   Sonos speakers
+Bose	   Bose speakers
+Wemo	   Wemo switch
+           Wemo motion sensor
+Hue		   Hue
+========== ======================================
```
Comment (jimmyjames) [inline cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst]: I wonder if we need to be more specific about which devices/models specifically are included. Probably should check with the KB article draft to see if they list specific model types, and use that.

Comment (rajkaramchedu) [inline cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst]: Will check.

Comment (rajkaramchedu) [inline cloud-and-lan-connected-device-types-developers-guide/super-lan-connect.rst]: Refer to #587. This PR is closed due to merge conflicts.

**Comment by rajkaramchedu** [comment]
Refer to #587. This PR is closed due to merge conflicts.

**Pull request closed without merging**